descriptors/ModuleDescriptor-template.json

```diff
@@ -270,6 +270,18 @@
             "marccat.authority.insert.post"
           ]
         },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/marccat/authority-record/{id}",
+          "modulePermissions": [
```

Comment:
Reformat like this:
 "permissionsRequired": ["marccat.authority-record.item.get" ]

Reply:
Changed